src/plugins/intel_cpu/src/scratch_pad.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "common/memory.hpp"
+#include <memory>
+#include "cpu_memory.h"
+#include "dnnl_extension_utils.h"
+
+namespace ov {
+namespace intel_cpu {
+
+class ScratchPad {
```

Comment:
That's a little bit pedantic, but we are trying to distinguish between pure CPU plugin and oneDNN related functionality by adding DNNL prefix (for classes, files etc). This scratchpad is only suitable for oneDNN based modes and cannot be used by CPU plugin impls. Let's highlight that,

Reply:
changed